plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -660,6 +662,28 @@ private void createAndPopulateNationAndRegionData(TestingKafka kafka, TestingPin
         kafka.sendMessages(nationRowsBuilder.build().stream(), schemaRegistryAwareProducer(kafka));
         pinot.createSchema(getClass().getClassLoader().getResourceAsStream("nation_schema.json"), nationTableName);
         pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("nation_realtimeSpec.json"), nationTableName);
+
+        // Ensure two tables are available in Pinot with expected number of rows.
```

Comment:
I'm not sure if the cause of #14277 is preparation failure. Could you send another PR for stress test without this change?

Reply:
Sent #14308

Reply:
Thanks for adding this! The CI failed is : https://github.com/trinodb/trino/actions/runs/3127676861/jobs/5075173404

Reply:
Thanks! We still want to run stress tests without this change to make sure the failure rate. If #14308 doesn't fail, it means 50 attempts in this PR were insufficient.

Reply:
hmm, so no flaky? Then I cannot think of how this could happen

Reply:
Yeah, another 50 rounds passed: https://github.com/trinodb/trino/actions/runs/3133972403/jobs/5088009373